### DISCUSSION TITLE
TAB selection doesn't work

### DISCUSSION THREAD

**Discussion by Bedtimestory9** [ibhagwan/fzf-lua#1170] [Q&A] [answered]
Hi, thanks for creating fzf-lua

But I'm having a hard time to get TAB selection to work in fzf-lua, it only applies quickfix marker to each selection, and when I press ENTER on the highlighted selection with TAB it opens the quickfixes. I wish TAB could just function like normal autocomplete select.

**Comment by ibhagwan** [comment]
> I wish TAB could just function like normal autocomplete select.

I don’t understand what this means, fzf is a terminal process, the key binds are (mostly) fzf keybinds, there is no auto completion in fzf in terminal mode.

**Reply by Bedtimestory9** [reply]
Sorry for the confusions.
I mean if I type something in fzf-lua, results appears, and then if I wanna select, for example, 2nd result, using tab, and press Enter, it will go to the first result instead. It will never go to the file where I highlighted with Tab, it only works with arrow keys, which will go to the highlighted result as expected.
How to make tab select work?

**Reply by ibhagwan** [reply]
No such option unfortunately, you can disable multi select with `--multi` but I don’t think that’s what you’re after, this is controlled by fzf.

**Reply by ibhagwan** [reply, marked as answer]
If you want you can make tab behave like arrows, but you’re gonna lose the ability to select multiple entries (and send these to the quickfix list).

if that’s what you’re after take a look at the default options section in the README, search for the `keymap.fzf` table and rebind tab to `down`.

**Reply by Bedtimestory9** [reply]
Through your answer, I was looking at the default option and ran into a line that remap C-j to Down, so fzf has built-in vim like selection after all and I was just a bit too clueless to think of it. Either way, if I haven't stumble across it I would've remapped Tab to Down too cuz that was what I wanted as well. Thanks.

**Reply by ibhagwan** [reply]
Great!

**Comment by jrock2004** [comment]
@Bedtimestory9 did you find a solution? If so could you show me a code snippet of what you did?
